src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family for the case when the 'Roboto' font is not available. This ensures text is still displayed with a similar style font, for example: `font-family: Roboto, Arial, sans-serif;`.